packages/fx-core/resource/package.nls.json

```diff
@@ -385,6 +385,7 @@
   "error.NoSubscriptionFound": "Failed to find a subscription.",
   "error.TrustCertificateCancelError": "User canceled. For Teams to trust the self-signed SSL certificate used by the toolkit, a self-signed certificate must be added to your certificate store.",
   "error.NoTeamsAppTenantId": "Cannot find Teams app tenant id",
+  "error.NoAppStudioToken": "Cannot find App Studio token",
```

Comment:
Is it still officially called App Studio? Please confirm the new name. The frontend of App Studio has been renamed to Teams Developer Portal.